openslides_backend/action/actions/user/create.py

```diff
@@ -67,6 +70,10 @@ class UserCreate(
     history_information = "Account created"
     own_history_information_first = True
 
+    def check_permissions(self, instance: dict[str, Any]) -> None:
+        self.assert_not_anonymous()
```

Comment:
Ensure there's a test that checks this line of code.
Then delete this line and see if it still works. Because AFAIK anonymous should be auto-locked out of action requests by now.